scripts/gulpfiles/build_tasks.js

```diff
@@ -672,11 +688,10 @@ const build = gulp.parallel(
  */
 function checkinBuilt() {
   return gulp.src([
-    `${BUILD_DIR}/**.js`,
-    `${BUILD_DIR}/**.js.map`,
-    `${BUILD_DIR}/**/**.js`,
-    `${BUILD_DIR}/**/**.js.map`,
-  ]).pipe(gulp.dest('.'));
+    `${BUILD_DIR}/*_compressed.js`,
+    `${BUILD_DIR}/*_compressed.js.map`,
+    `${BUILD_DIR}/msg/js/*.js`,
+  ], {base: BUILD_DIR}).pipe(gulp.dest('.'));
```

Comment:
Does this put the messages correctly in the msg folder? Or does it dump all of them into the root directory?

Reply:
A question I learned the answer to the hard way (oh look my Blockly directory is suddenly full of spam!), so now the `{base: BUILD_DIR}` option is there to tell `gulp.dest` not to trim off anything more than `build/` from the output path for (any of, but particularly) the messages files.